lsstswBuild.sh

```diff
@@ -7,12 +7,15 @@
 #  expectations regarding the 'work' directory location are  equivalent.
 # /\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\
 
+set -e
+
 SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
```

Comment:
That is the intent although I confess to cargo culting it at this point.  Alternatives are to use `readlink`, which I know has different flags between gnu and bsd or `${BASH_SOURCE[0]}`.  The later is probably the best option if truly equivalent as it avoids shelling out.